chart/templates/service.yaml

```diff
@@ -0,0 +1,23 @@
+apiVersion: v1
+kind: Service
+metadata:
+  name: {{ include "keess.fullname" . }}
+  namespace: {{ .Release.Namespace }}
+  labels:
+    {{- include "keess.labels" . | nindent 4 }}
+  annotations:
+    prometheus.io/scrape: "true"
+    prometheus.io/port: "{{ .Values.service.port }}"
+    prometheus.io/path: "/metrics"
```

Comment:
Since you are already creating the serviceMonitor, you don't need to pass this annotations in the service, but at the same time it doesn't have any downside, unless  you assume that by disabling serviceMonitor you would expect that prometheus wouldn't scrape keess metrics. 
```yaml
metrics:
  serviceMonitor:
    enabled: false
```

In this case it would still continue even without the serviceMonitor because this service would still be created with the annotations.